docker/pytorch-aarch64/scripts/build-pytorch-xla.sh

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+
+# *******************************************************************************
+# Copyright 2020-2022 Arm Limited and affiliates.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# *******************************************************************************
+
+set -euo pipefail
+
+cd $PACKAGE_DIR
+readonly package=pytorch
+readonly version=$TORCH_VERSION
+readonly src_host=https://github.com/pytorch
+readonly src_repo=pytorch
+readonly num_cpus=$(grep -c ^processor /proc/cpuinfo)
+
+# Clone PyTorch and checkout the commit that is compatible with xla
+# Checking out the latest commit as of June 26th, this is to get the latest
+# PyTorch with channels_last format support. All the PR patches have been
+# rebased to these commits
+git clone ${src_host}/${src_repo}.git
+cd ${src_repo}
+git checkout 8a5cc940e39820ad61dedee5c01f366af182ff3a
```

Comment:
As I mentioned above, this build option is for downstream changes along with pytorch stable master commit. I left the torch version still as it is used for sanity checking the wheel version. but agree with you, it will be misleading as the torch version shows 1.11 (since thats what provided to the build) but ideally this is 1.13+gitcommit. I will clean it up